code/script.js

```diff
@@ -1,47 +1,131 @@
 // All the DOM selectors stored as short variables
 const chat = document.getElementById('chat')
-
-// Global variables, if you need any, declared here
-
-// Functions declared here
+const inputWrapper = document.getElementById('input-wrapper')
+const form = document.getElementById('name-form')
+const nameInput = document.getElementById('name-input')
+const messageAudio = new Audio ("assets/click.wav")
+const airplaneSound = new Audio ("assets/Airplanesound.wav")
 
 // This function will add a chat bubble in the correct place based on who the sender is
-const showMessage = (message, sender) => {
+const showMessage = (message, sender) => { 
   if (sender === 'user') {
     chat.innerHTML += `
       <section class="user-msg">
         <div class="bubble user-bubble">
           <p>${message}</p>
         </div>
-        <img src="assets/user.png" alt="User" />  
+        <img src="assets/user2.png" alt="User" />
+        
       </section>
     `
   } else if (sender === 'bot') {
     chat.innerHTML += `
       <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
+        <img src="assets/airplane.png" alt="Bot" />
         <div class="bubble bot-bubble">
           <p>${message}</p>
         </div>
       </section>
     `
   }
+  // Click sound when message is sent 
+  messageAudio.currentTime = 0
+  messageAudio.play()
+
   // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
   chat.scrollTop = chat.scrollHeight
 }
 
 // Starts here
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  showMessage(`Hey you, What's your name?`, 'bot')
+  
+}
+// Function that saves user's name
+const handleNameInput = (event) => {
+  event.preventDefault()
+  let name = nameInput.value
+  // User name will be 'stranger' if there is no input
```

Comment:
Think this comment is helpful to understand the two outcomes of the user reply (just like the rest of your comments in the JS-file)